app/controllers/orders_controller.rb

```diff
@@ -115,6 +115,8 @@ def add
           @order.invalidate! ## this is because we just added an order_detail
         rescue NUCore::MixedFacilityCart
           @order.errors.add(:base, "You can not add a product from another facility; please clear your cart or place a separate order.")
+        rescue NUCore::MixedBillingMode
+          @order.errors.add(:base, "You can mix billing modes with a non-billable product")
```

Comment:
I don't think users will know what this means, I wonder if we should instruct them to contact someone at the facility?  Maybe give them a support email mailto link?
```suggestion
          @order.errors.add(:base, "You can not mix billing modes with a non-billable product")
```